docs/src/tutorial/minimal_working_compute.jl

```diff
@@ -0,0 +1,266 @@
+                            SHADER_STAGE_COMPUTE_BIT,
+                            shader,
+                            "main", # this needs to match the function name in the shader
+                            specialization_info = SpecializationInfo(
+                                [SpecializationMapEntry(0, 0, 4)],
+                                UInt64(4),
+                                Ptr{Nothing}(pointer(spec_consts)),
+                            ),
+                        ),
+                        pl,
+                        -1,
+                    ),
+                ],
+            ),
+        ),
+    ),
+)
+
+# Now make a descriptor pool that is used to allocate the buffer descriptors
+# from (not a big one, just 1 descriptor set and total 1 descriptor)
+dpool = unwrap(create_descriptor_pool(device, 1, [DescriptorPoolSize(DESCRIPTOR_TYPE_STORAGE_BUFFER, 1)]))
+
+# Allocate the descriptors for our layout
+dsets = unwrap(allocate_descriptor_sets(device, DescriptorSetAllocateInfo(dpool, [dsl])))
+dset = first(dsets)
+
+# Make the descriptors point to the right buffers
+update_descriptor_sets(
+    device,
+    [
+        WriteDescriptorSet(
+            dset,
+            0,
+            0,
+            DESCRIPTOR_TYPE_STORAGE_BUFFER,
+            [],
+            [DescriptorBufferInfo(buffer, 0, WHOLE_SIZE)],
+            [],
+        ),
+    ],
+    [],
+)
+
+# ## Executing the shader
+#
+# Let's create a command pool in the right queue family, and take a command
+# buffer out of that.
+cmdpool = unwrap(create_command_pool(device, qfam_idx))
+cbufs = unwrap(allocate_command_buffers(device, CommandBufferAllocateInfo(cmdpool, COMMAND_BUFFER_LEVEL_PRIMARY, 1)))
+cbuf = first(cbufs)
+
+# Now that we have a command buffer, we can fill it with commands that cause
+# the kernel to be run. Basically, we bind and fill everything, and then
+# dispatch a sufficient amount of invocations of the shader to span over the
+# array.
+begin_command_buffer(cbuf, CommandBufferBeginInfo(flags = COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT))
+
+cmd_bind_pipeline(cbuf, PIPELINE_BIND_POINT_COMPUTE, p)
+
+const_buf = [shader_push_consts(1.234, data_items)]
+cmd_push_constants(cbuf, pl, SHADER_STAGE_COMPUTE_BIT, 0, sizeof(shader_push_consts), Ptr{Nothing}(pointer(const_buf)))
+
+cmd_bind_descriptor_sets(cbuf, PIPELINE_BIND_POINT_COMPUTE, pl, 0, [dset], [])
+
+cmd_dispatch(cbuf, div(data_items, const_local_size_x, RoundUp), 1, 1)
+
+end_command_buffer(cbuf)
+
+# Finanly, find a handle to the compute queue and send the command to execute the shader!
+compute_q = get_device_queue(device, qfam_idx, 0)
+queue_submit(compute_q, [SubmitInfo([], [], [cbuf], [])])
+
+# ## Getting the data
+#
+# Now, data is being crunched in the background. To get the data, we need to
+# wait for completion and invalidate the mapped memory (so that whatever
+# happened on the GPU gets to the mapped range visible for the "host" device.
```

Comment:
```suggestion
# Now, data is being crunched in the background. To get the data, we need to
# wait for completion and invalidate the mapped memory (so that whatever
# happened on the GPU gets to the mapped range visible for the "host" device).
```